### ISSUE TITLE
BioBERT base model files

### ISSUE THREAD

**Issue by antonyscerri** [dmis-lab/biobert#8]
Hi

I had a question regarding the base bioBERT model files you have released, and some of the file sizes. I downloaded the "pubmed_pmc_470k" set and noted the checkpoint file (biobert_model.ckpt.data-00000-of-00001) is ~1.3G. This looks more like the size seen with the large BERT models compared to the ~440MB small base BERT models (the multilingual ones appeared to be about ~700MB. The config file and vocab files appear to match with what was seen in the smaller BERT base model. I was curious incase the file included fine tuning on one of the tasks or really the base bioBERT, i'm pretty sure it doesn't and it looks like they might just contain the optimizer parameters.

Could you also confirm whether you pre-trained bioBERT from scratch using the various corpora, or did you initialise it with one of the BERT base models and only continued pre-training with the biomedical corpora (if so could please indicate exactly which one, I don't believe you have used a multilingual based on vocab etc). One other thing you may wish to consider adding to your paper is the difference between case and uncased variants, whether you tested this or not and that you are using the cased variant at the moment it seems.

Thanks again

Tony

**Comment by jhyuklee** [comment]
We used cased bert base model (not multilingual), and will check the file size again, but it perfectly works as bert base model. And as mentioned in the paper, we initialized with the original bert, and only continued pretraining with bio corpora. Cased version was used because most previous works preferred cased tokenization, and it seems natural for bio domain to use cased words as it contains lots of proper nouns. We didn’t test uncased version due to limitations on GPUs.
